cl_dll/hud.h

```diff
@@ -590,6 +590,7 @@ class CHud
 	int m_iFontHeight;
 
 	cvar_t* m_pCvarColor;
+	cvar_t* cl_autostop;
```

Comment:
Could you move this to right below `m_pCvarDrawDeathNoticesAlways` and name it `m_pCvarAutostop`? Right now it sits right in the middle of two `hud_color`-related things and its name is inconsistent with the others.